templates/components/search/controls.html.twig

```diff
@@ -101,7 +101,7 @@
                <i id="export_dropdown_icon" class="ti fa-lg ti-file-download"></i>
             </span>
          </button>
-      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&start=' ~ start ~ '&' ~ posthref %}
+      {% set exporthref = path('/front/report.dynamic.php') ~ '?item_type=' ~ itemtype ~ '&sort[]=' ~ sort[0] ~ '&order[]=' ~ order[0] ~ '&start=' ~ start ~ '&' ~ posthref %}
```

Comment:
```suggestion
      {% set exporthref = path('/front/report.dynamic.php') ~ {
        'item_type': itemtype,
        'sort': sort,
        'order': order,
        'start': start,
      }|url_encode ~ posthref %}
```

This to have also multisort feature included in the href

Reply:
ping @yungtechboy1 ?

Reply:
This works also, just need 1 small change...